test/integration/setup/adminclient/client.go

```diff
@@ -368,6 +371,19 @@ func batchSetFederatedBundle(ctx context.Context, c *itclient.Client) error {
 	return nil
 }
 
+func countFederatedBundles(ctx context.Context, c *itclient.Client) error {
```

Comment:
in all places it is called countBundles, may we call the same here?

Reply:
Thanks; cut/paste error